web_src/less/_user.less

```diff
@@ -169,6 +169,21 @@
   }
 }
 
+.user-badges {
+  display: flex;
+  flex-flow: row wrap;
+  padding: 0;
+  margin: -3px !important;
+
+  li {
+    display: flex;
+    border-bottom: 0 !important;
+    padding: 3px !important;
+    width: 20%;
+    max-width: 60px;
+  }
+}
+
```

Comment:
As I said, we should use a static badge size of for example 64px, which should also be set on `<img>` as width and height. Either flexbox with `flex-wrap` or even better, `display: grid` will work to distribute the items:

```css
.user-badges {
  display: grid;
  grid-template-columns: repeat(auto-fill, 64px);
  gap: 2px;
}

Reply:
updated, thanks for this suggestion.

Reply:
Please test it with a few badges so that they wrap, I didn't actually test it myself :)